Makefile

```diff
@@ -146,3 +146,10 @@ release-skip-tests quick-release:
 	KUBE_RELEASE_RUN_TESTS=n KUBE_FASTBUILD=true build/release.sh
 .PHONY: release-skip-tests quick-release
 
+# Build ginkgo for tests
+#
+# Example:
+#   make ginkgo
+ginkgo:
+	hack/build-go.sh vendor/github.com/onsi/ginkgo/ginkgo
+.PHONY: ginkgo
```

Comment:
Yeah. `hack/build-go.sh` will set GOBIN to `_output`.

On Wed, May 25, 2016 at 11:35 AM, Tim St. Clair <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/kubernetes/kubernetes/pull/26228#discussion_r64627539>
> :
>
> > @@ -146,3 +146,10 @@ release-skip-tests quick-release:
> >  	KUBE_RELEASE_RUN_TESTS=n KUBE_FASTBUILD=true build/release.sh
> >  .PHONY: release-skip-tests quick-release
> >
> > +# Build ginkgo for tests
> > +#
> > +# Example:
> > +#   make ginkgo
> > +ginkgo:
> > +	hack/build-go.sh vendor/github.com/onsi/ginkgo/ginkgo
> > +.PHONY: ginkgo
>
> nvm - I assume it will be in the _output directory?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26228/files/a07c9833fc0cc42e0b93f3835b7bb8e86d7843f7#r64627539>
>